sherpa/sim/sample.py

```diff
@@ -119,7 +122,8 @@ def get_scales(self, fit, myscales=None):
 
             covar = Covariance()
             covar.config['sigma'] = self.sigma
-            fit.estmethod = Covariance()
```

Comment:
Hmmm, this looks like a logical error in out code. It's fixed in this PR but we should check.